packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -298,14 +306,42 @@ export class PendingStateManager implements IDisposable {
 	 * that the batch information is correct.
 	 * @param batch - The batch that is being processed.
 	 * @param batchStartCsn - The clientSequenceNumber of the start of this message's batch
+	 * @param sequenceNumber - The sequence number of the batch message.
```

Comment:
```suggestion
	 * @param sequenceNumber - If the batch is empty, the sequence number of the empty batch placeholder message. Otherwise, undefined.
```